proposals/nnnn-Use-Taskmaster-To-Handle-Queuing-Operations-In-Managers.md

```diff
@@ -0,0 +1,80 @@
+# Use Taskmaster To Handle Queuing Operations in Managers
```

Comment:
```suggestion
# Use TaskMaster To Handle Queuing Operations in Managers
```

Reply:
Taskmaster seems to be one word in English. I will update the word in all places for consistency.